venv/Lib/site-packages/_distutils_hack/__init__.py

```diff
@@ -0,0 +1,222 @@
+# don't import any costly modules
```

Comment:
Files under the venv directory belong to the Python development environment and should not be committed.